src/components/Edge-Conn-PCIe.stanza

```diff
@@ -0,0 +1,943 @@
+#use-added-syntax(jitx)
+defpackage connectors/components/Edge-Conn-PCIe :
+  import core
+  import jitx
+  import jitx/commands
+  import jsl
+
+pcb-pad rect-connect-pad-1 :
+  type = SMD
+  shape = Rectangle(0.700, 3.200)
+  layer(SolderMask(Top)) = Rectangle(0.700, 3.200)
+
+pcb-pad rect-connect-pad-2 :
+  type = SMD
+  shape = Rectangle(0.700, 4.300)
+  layer(SolderMask(Top)) = Rectangle(0.700, 4.300)
+
+public pcb-landpattern LP-PCIe-Edge (width : Int) :
+
+  pad A1 : rect-connect-pad-1 at loc(0.0, 0.550) on Bottom
+  pad A2 : rect-connect-pad-2 at loc(1.000, 0.0) on Bottom
+  pad A3 : rect-connect-pad-2 at loc(2.000, 0.0) on Bottom
```

Comment:
Was this file auto generated ? This is a lot of copy/paste.

Reply:
It started life as an imported component. I thought about making it cleaner, but the cost-benefit seemed low.